facefusion/temp_helper.py

```diff
@@ -1,50 +1,47 @@
-	temp_directory_path = get_temp_directory_path(file_path)
+def get_temp_file_path(temp_path : str, file_path : str) -> str:
+	temp_directory_path = get_temp_directory_path(temp_path, file_path)
 	temp_file_extension = get_file_extension(file_path)
 	return os.path.join(temp_directory_path, 'temp' + temp_file_extension)
 
 
-def move_temp_file(move_path : str) -> bool:
-	temp_file_path = get_temp_file_path(move_path)
+def move_temp_file(temp_path : str, move_path : str) -> bool:
+	temp_file_path = get_temp_file_path(temp_path, move_path)
 	return move_file(temp_file_path, move_path)
 
 
-def resolve_temp_frame_paths(target_path : str) -> List[str]:
-	temp_frames_pattern = get_temp_frames_pattern(target_path, '*')
+def resolve_temp_frame_paths(temp_path : str, target_path : str, temp_frame_format : str) -> List[str]:
+	temp_frames_pattern = get_temp_frames_pattern(temp_path, target_path, '*', temp_frame_format)
 	return resolve_file_pattern(temp_frames_pattern)
 
 
-def get_temp_frames_pattern(target_path : str, temp_frame_prefix : str) -> str:
-	temp_directory_path = get_temp_directory_path(target_path)
-	return os.path.join(temp_directory_path, temp_frame_prefix + '.' + state_manager.get_item('temp_frame_format')) # TODO: remove state_manager.get
+def get_temp_frames_pattern(temp_path : str, target_path : str, temp_frame_prefix : str, temp_frame_format : str) -> str:
+	temp_directory_path = get_temp_directory_path(temp_path, target_path)
+	return os.path.join(temp_directory_path, temp_frame_prefix + '.' + temp_frame_format)
 
 
-def get_temp_directory_path(file_path : str) -> str:
+def get_temp_directory_path(temp_path : str, file_path : str) -> str:
 	temp_file_name = get_file_name(file_path)
-	return os.path.join(state_manager.get_item('temp_path'), 'facefusion', temp_file_name) # TODO: remove state_manager.get
+	return os.path.join(temp_path, 'facefusion', temp_file_name)
 
 
-def create_temp_directory(file_path : str) -> bool:
-	temp_directory_path = get_temp_directory_path(file_path)
+def create_temp_directory(temp_path : str, file_path : str) -> bool:
+	temp_directory_path = get_temp_directory_path(temp_path, file_path)
 	return create_directory(temp_directory_path)
 
 
-def clear_temp_directory(file_path : str) -> bool:
-	if not state_manager.get_item('keep_temp'): # TODO: remove state_manager.get
-		temp_directory_path = get_temp_directory_path(file_path)
-		return remove_directory(temp_directory_path)
-	return True
+def clear_temp_directory(temp_path : str, file_path : str) -> bool:
+	temp_directory_path = get_temp_directory_path(temp_path, file_path)
+	return remove_directory(temp_directory_path)
 
 
-def get_temp_sequence_paths(file_path : str, frame_total : int, temp_frame_prefix : str, temp_frame_format : str) -> List[str]:
-	temp_directory_path = get_temp_directory_path(file_path)
+def get_temp_sequence_paths(temp_path : str, file_path : str, frame_total : int, temp_frame_prefix : str, temp_frame_format : str) -> List[str]:
```

Comment:
look at the other methods... different ordering of args here - frame_total to the end?